src/main/scala/io/iohk/ethereum/blockchain/sync/FastSyncController.scala

```diff
@@ -114,7 +114,8 @@ class FastSyncController(
           log.info("Received target block from peer, starting fast sync")
 
           scheduler.schedule(0.seconds, printStatusInterval, self, PrintStatus)
-          context become new SyncingHandler(targetBlockHeader).receive
+
+          context become new SyncingHandler(targetBlockHeader, fastSyncStateStorage).receive
```

Comment:
you don't have to pass `fastSyncStateStorage` there. it's already accessible in whole `FastSyncController`